File: DarkOrbit 10.0/Net/netty/handlers/AudioSettingsRequestHandler.cs
using Ow.Game;
using Ow.Managers;
using Ow.Net.netty.commands;
using Ow.Net.netty.requests;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ow.Net.netty.handlers
{
    class AudioSettingsRequestHandler : IHandler
    {
        public void execute(GameSession gameSession, byte[] bytes)
        {
            var read = new AudioSettingsRequest();
            read.readCommand(bytes);

            var player = gameSession.Player;
            var audioSettings = player.Settings.Audio;

            audioSettings.notSet = false;
            audioSettings.music = read.music;
            audioSettings.sound = read.sound;
            audioSettings.voice = read.voice;
            audioSettings.playCombatMusic = read.playCombatMusic;

            QueryManager.SavePlayer.Settings(player, "audio", audioSettings);
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/handlers/AudioSettingsRequestHandler.java
package simulator.netty.handlers;

import mysql.QueryManager;
import net.game_server.GameServerClientConnection;
import net.game_server.GameSession;
import simulator.netty.ClientCommand;
import simulator.netty.clientCommands.AudioSettingsRequest;
import simulator.users.ClientSettings;
import utils.Log;

/**
 Created by Pedro on 31-03-2015.
 */
public class AudioSettingsRequestHandler
        implements ICommandHandler {

    private final GameServerClientConnection mGameServerClientConnection;
    private final AudioSettingsRequest       mAudioSettingsRequest;

    public AudioSettingsRequestHandler(final GameServerClientConnection pGameServerClientConnection,
                                       final ClientCommand pAudioSettingsRequest) {
        this.mGameServerClientConnection = pGameServerClientConnection;
        this.mAudioSettingsRequest = (AudioSettingsRequest) pAudioSettingsRequest;
    }

    @Override
    public void execute() {
        Log.pt("AUDIO SETTINGS REQUEST HANDLER");
        final GameSession gameSession = this.mGameServerClientConnection.getGameSession();
        if (gameSession != null) {
            final ClientSettings.Audio audio = gameSession.getAccount()
                                                          .getClientSettingsManager()
                                                          .getClientSettings()
                                                          .getAudio();
            audio.setNotSet(false);
            audio.setMusic(this.mAudioSettingsRequest.music);
            audio.setSound(this.mAudioSettingsRequest.sound);
            audio.setVoice(this.mAudioSettingsRequest.voice);
            audio.setPlayCombatMusic(this.mAudioSettingsRequest.playCombatMusic);
            QueryManager.saveAccount(gameSession.getAccount());
        }
    }
}
